youtube_dl/extractor/youtube.py

```diff
@@ -1914,9 +1943,50 @@ def _real_extract(self, url):
             player_response = self._extract_yt_initial_variable(
                 webpage, self._YT_INITIAL_PLAYER_RESPONSE_RE,
                 video_id, 'initial player response')
-        if not player_response:
+        if False and not player_response:
```

Comment:
The "False and" looks like a temporary stopgap. It could thus use an inline comment explaining where to read about  what problem it mitigates and what work is needed to fix it properly. Ideally the explanation would be a link to an issue thread.